### DISCUSSION TITLE
FreeBSD Compatibility

### DISCUSSION THREAD

**Discussion by cyliamargatroid** [ikemen-engine/Ikemen-GO#826] [Feature Request]
Is it Possible if you can Port IKEMEN to FreeBSD?

**Comment by cyliamargatroid** [comment]
I was going to Build IKEMEN GO on FreeBSD, I really need some help with FreeBSD Version of IKEMEN.

**Comment by cyliamargatroid** [comment]
Probably go-openal needs to have FreeBSD compatibility

**Comment by Windblade-GR01** [comment]
I'll check it latter.
But we are limited by the compatibility of the libraries.

The main 3 libraries are OpenAL (Audio), GLFW (Input) and OpenGL (Video)

**Comment by cyliamargatroid** [comment]
ok.
